diff --git a/dist/index.js b/dist/index.js
index 9de2bd4c3d9820b371825b4025991be48cabc3ea..6daaff0e0e9d6eb583bf223e8cc92c80876cb896 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -1007,7 +1007,7 @@ class OpenAI extends ToolCallLLM {
         return this._live;
     }
     get metadata() {
-        const contextWindow = ALL_AVAILABLE_OPENAI_MODELS[this.model]?.contextWindow ?? 1024;
+        const contextWindow = ALL_AVAILABLE_OPENAI_MODELS[this.model]?.contextWindow ?? 200000;
         return {
             model: this.model,
             temperature: this.temperature,
